laminci/nox.py

```diff
@@ -1,20 +1,20 @@
 import json
 import os
 import shlex
+from collections.abc import Iterable
 from pathlib import Path
-from typing import Dict, Iterable, Literal, Optional, Union
+from typing import Literal, Optional, Union
 
 import nox
 from nox import Session
 
-from . import _nox_logger  # noqa, the import silences the logger
```

Comment:
This here broke all our CI logging @Zethson in the sense that it's pure clutter now 😅 

Why did ruff remove this even though it's decorated with noqa?

Do you look through changes that ruff makes?

Reply:
Hm, pre-commit didn't even remove it: https://github.com/laminlabs/laminci/commit/5db8b765d4fee0bf958f7f7c567642fe1959275c

Why did it get removed here then? Is there an in-concistent setup between pre-commit and what you presumably ran on all files?

Reply:
Meaning: did you _not_ run `pre-commit run --all-files` but somehow manually ruff with a slightly different set of options?

Reply:
I absolutely did not run ruff with a slightly different set of options. I don't know how or why that happened. I must've missed this while looking at the changed lines, sorry.

Reply:
Then it's all good but still weird! ruff did not auto-remove this line when I ran pre-commit this morning. 🤷